### PR TITLE
default project resource quota docs

### PR DESCRIPTION
**Pull request by lsviben**
Added some info about default project resource quotas

### CODE REVIEW

**Comment by lsviben** [comment]
/hold
until the code gets merged

**Comment by embik** [comment]
@lsviben the corresponding PR (https://github.com/kubermatic/kubermatic/pull/11582) got merged, so I think this no longer needs to be on hold, no?

**Comment by lsviben** [comment]
> @lsviben the corresponding PR ([kubermatic/kubermatic#11582](https://github.com/kubermatic/kubermatic/pull/11582)) got merged, so I think this no longer needs to be on hold, no?

Right. Somehow I thought this got merged already

**Review by embik — APPROVED** [review]

**Pull request merged**
